### ISSUE TITLE
Dev.Portal: Drop-down Menus not working in Windows Chrome

### ISSUE THREAD

**Issue by EnTerr** [rokudev/docs#9]
Steps to reproduce:
1. Start Chrome (v.53.0.2785.143 m 64-bit in my case) on WIndows (10 in my case)
2. Open https://developer.roku.com/developer-channels/channels (login and what-have-you)
3. Click on "Manage My Channels ^" link in upper-left - drop-down menu opens
4. Try clicking on any of the links in that menu ("Developer Home", "Manage My Channels" and so on) - **nothing happens, they are not clickable**.  

The drop-down menus also don't work in any subscreens, e.g.
https://developer.roku.com/developer-channels/channels/edit/package/<99999>

This happens on vanilla Windows (no bloatware) and Chrome (no extensions).
I see nothing suspicious in browser's JS console.

The menu works in Chrome on Mac and IE for WIndows - so seems the bug is specific to Chrome+Windows - also issue is not PEBKAC.

**Comment by EnTerr** [comment]
Per @belltown, also broken in Chrome or Firefox on Ubuntu, 
see https://forums.roku.com/viewtopic.php?f=34&t=96634&start=15#p541507

**Comment by jr-roku** [comment]
@EnTerr Hi, we have not been able to reproduce the problem you noted at the top of this ticket. Not sure how to proceed. We can see from backend logs that the usage is normal on windows and only have this one report of it not working. The issue reported by @belltown affects the navigation bar on developer.roku.com (not the drop-down intra-page navigation). That one is confirmed and will be fixed shortly.

**Comment by EnTerr** [comment]
@jr-roku i am puzzled as to why you are unable to reproduce the issue! Please note this is not a one-off report - the same thread i linked has the same complaint by another experienced developer, for **Chrome on Windows**:

> [TheEndless wrote:](https://forums.roku.com/viewtopic.php?f=34&t=96634#p541500) 
> Has anyone had any luck getting the dropdowns on the new dev portal to work in Google Chrome?  > Without those working, the site is pretty much useless in Chrome, which is beyond frustrating.

In an effort to help you troubleshoot, i just did an additional test:

- On a completely different Windows machine (surface book w/64bit Windows 10 Pro)
- On a different LAN
- With a different Chrome installation (53.0.2785.143m 64bit)
- In "Incognito" mode, as to exclude potential chrome-user-login settings
- With extensions disabled (Menu > More tools > Extensions)
- Using completely different Roku developer account

and **the issue is still reproducible!**

So to recap, as a reason for the glitch i have excluded: single person, single developer account, single windows installation, single chrome installation, single gmail account, single network issue, extension interference. Therefore the issue is reliably reproducible.

How can i help you further to see it?!
Would chatting up me or @theEndless help? Ask the support engineers for our contact info.

**Comment by belltown** [comment]
The **Manage My Channels ^** link works for me in Firefox under Ubuntu, but not using Chrome.

**Comment by jr-roku** [comment]
@EnTerr I'm sorry you are still having problems. We do our testing in VMs so it could be that it is not showing up there. Can you email me directly on jroberts AT roku.com ? Thanks

**Comment by jr-roku** [comment]
This issue should have been fixed now. Please LMK if there are still problems...

**Comment by EnTerr** [comment]
Confirming - the issue seems fixed now.
There is a minor oddity in the menu staying open if nothing is selected - or being able to open both drop-down menus at once - but that's peanuts. The important part is now it works. 
This one was toughie for you to reproduce and troubleshoot - thank you for the email guidance. 
Cheers!
